validator/validator-main.protoascii

```diff
@@ -1537,13 +1537,25 @@ tags: {
   tag_name: "ARTICLE"
 }
 # 4.3.3 The section element
+attr_lists: {
+  name: "poool-access-attrs"
+  attrs: {
+    name: "poool-access-preview"
+    requires_extension: "amp-access-poool"
+  }
+  attrs: {
+    name: "poool-access-content"
+    requires_extension: "amp-access-poool"
+  }
+}
 tags: {
   html_format: AMP
   html_format: AMP4ADS
   html_format: AMP4EMAIL
   html_format: ACTIONS
   tag_name: "SECTION"
   disallowed_ancestor: "AMP-ACCORDION"
+  attr_lists: "poool-access-attrs"
```

Comment:
Do you also want this to apply to `section`s in amp-accordion?

https://github.com/ampproject/amphtml/blob/master/extensions/amp-accordion/validator-amp-accordion.protoascii#L60

If so, add this one line there too. If not, all good.

Reply:
I don't think we need it right now, having your `amp-access` preview/full content inside an `amp-accordion` would be quite weird anyway. If we ever need it, I will make another PR :)